app/build.gradle

```
@@ -88,4 +89,11 @@ dependencies {
     // Glide
     implementation "com.github.bumptech.glide:glide:$glide_version"
     annotationProcessor "com.github.bumptech.glide:compiler:$glide_version"
+
+    // Retrofit2
+    implementation 'com.squareup.retrofit2:retrofit:2.9.0'
+    implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
```

중요한 건 아니지만, 두 의존성 버전은 계속 동일하게 갈 가능성이 높으니 묶어둘까요 ?
```suggestion
    // Retrofit2
    def retrofit_version = '2.9.0'
    implementation "com.squareup.retrofit2:$retrofit_version"
    implementation "com.squareup.retrofit2:converter-gson:$retrofit_version"
```

---

반영했습니다!